app/views/posts/index.html.erb

```diff
@@ -11,6 +11,9 @@
       <h3>Post #<%= post.id %></h3>
       <p><%= post.text %></p>
       <h4>Comments <%= post.comments_counter %> likes: <%= post.likes_counter %></h4>
+       <%= form_with url: "/users/#{@user.id}/posts/#{post.id}/likes" do |f| %>
+        <%= f.submit "like posts"%>
+      <% end %>
```

Comment:
- Please not that when I populated the database to see the form created, I'm met with an empty page. Kindly look into this 👍🏽
![Screenshot from 2023-02-04 00-41-23](https://user-images.githubusercontent.com/44624138/216732240-aae51418-8aeb-49d5-9ee5-ae27331bc164.png)